lib/jobs.js

```diff
@@ -27,6 +27,10 @@ async function unzip(config) {
     logger.info(`Job:unzip started. buildId: ${config.buildId}`);
 
     try {
+        if (!Number.isInteger(PARALLEL_UPLOAD_LIMIT)) {
+            throw new Error('A value other than an integer is set for WORKER_PARALLEL_UPLOAD_LIMIT.');
```

Comment:
Is it better to validate this when the worker starting(just behind L11)?
Or you can put a default value in L11 if the setting value is not number instead of validate.

Reply:
thx.
> Is it better to validate this when the worker starting(just behind L11)?

Fixed it.
> you can put a default value in L11 if the setting value is not number instead of validate.

I think it is better to do this in error for unintended values, since the component could be killed by the liveness probe if an unintended value is set.